src/components/UserProfile.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import firebase from 'firebase/app';
+import 'firebase/auth';
+import styled from '@emotion/styled';
+import GoogleLoginButton from './GoogleLoginButton';
+
+function UserProfile() {
+  const [userInfo, setUserInfo] = useState(null);
```

Comment:
`~info` 같은 postfix가 꼭 필요하진 않은 것 같기는 합니다. 어차피 `user`라고만 해도 유저에대한 정보라서..

```suggestion
  const [user, setUser] = useState(null);
```